core/src/components/badge/test/hint/index.html

```diff
@@ -104,6 +104,136 @@
               <ion-badge color="danger" size="large" vertical="bottom"></ion-badge>
             </ion-avatar>
           </div>
+
+          <div class="ion-display-flex ion-align-items-center ion-justify-content-around ion-margin">
+            <ion-avatar size="xxsmall">
+              <img src="/src/components/avatar/test/avatar.svg" />
+              <ion-badge color="danger" size="small" vertical="top">1</ion-badge>
```

Comment:
Based on the design, we should be using `xxsmall`. Please change all the sizes to `xxsmall` instead.

```suggestion
              <ion-badge color="danger" size="xxsmall" vertical="top">1</ion-badge>
```

Reply:
Fixed